src/traces/parcoords/attributes.js

```diff
@@ -12,10 +12,12 @@ var colorAttributes = require('../../components/colorscale/color_attributes');
 var colorbarAttrs = require('../../components/colorbar/attributes');
 var colorscales = require('../../components/colorscale/scales');
 var axesAttrs = require('../../plots/cartesian/layout_attributes');
+var fontAttrs = require('../../plots/font_attributes');
 
 var extendDeep = require('../../lib/extend').extendDeep;
 var extendFlat = require('../../lib/extend').extendFlat;
 
+
```

Comment:
I think we're bikeshedding https://github.com/plotly/plotly.js/issues/950#issuecomment-298144207